code/game/objects/items/stacks/stack.dm

```diff
@@ -139,7 +139,7 @@
 		ui_interact(user)
 
 /obj/item/stack/attackby(obj/item/thing, mob/user, params)
-	if((parent_stack && !istype(thing, merge_type)) || (parent_stack && thing.type != type))
+	if((!parent_stack && !istype(thing, merge_type)) || (parent_stack && thing.type != type))
 		..()
 		return
```

Comment:
```suggestion
		return ..()
```

Reply:
There is no difference, just a style like on procs above

Reply:
it should be either return ..() or have no hanging return

Reply:
No hanging return means that the entire proc breaks again